File: Client/Client/models/Cinema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Client.models
{
    public class Cinema
    {
        private int cin_id;
        private string name;
        private string city;
            public int getCin_id()
            {
                return cin_id;
            }
            public void setCin_id(int cin_id)
        {
            this.cin_id = cin_id;
        }

            public string getName()
            {
                return name;
            }

            public void setName(string name)
            {
                this.name = name;
            }

            public string getCity()
            {
                return city;
            }

            public void setCity(string city)
            {
                this.city = city;
            }


        


    }
}

File: Hello1/src/java/model/Cinema.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

import java.util.Date;
import javax.xml.bind.annotation.XmlRootElement;

/**
 *
 * @author ngghe
 */
@XmlRootElement
public class Cinema {
    private  int cin_id ;
    private String name;
    private String city ;

    public int getCin_id() {
        return cin_id;
    }

    public void setCin_id(int cin_id) {
        this.cin_id = cin_id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    
}
